plasma_framework/contracts/src/framework/utils/ExitPriority.sol

```diff
@@ -8,11 +8,12 @@ library ExitPriority {
      * @dev Given a utxo position and a unique ID, returns an exit priority.
      * The combination of 'exitableAt' and 'txPos' is the priority for Plasma M(ore)VP protocol.
      * 'exitableAt' only provide granularity of block, thus add 'txPos' to provide priority of transaction.
+     * @notice Full explanation on fields' lengths can be found here: https://github.com/omisego/plasma-contracts/pull/303#discussion_r328850572
      * @param exitId Unique exit identifier.
      * @return An exit priority.
      *   Anatomy of returned value, most significant bits first
-     *   42 bits - timestamp (exitable_at); we can represent dates until 2109 
-     *   54 bits - blknum * 10^9 + txindex; to represent all transactions for 10 years we need only 54 bits
+     *   42 bits  - timestamp (exitable_at); we can represent dates until year 141431
+     *   54 bits  - blknum * CHILD_BLOCK_INTERVAL * 10^5 + txindex; 54 bits represent all transactions for 85 years.
```

Comment:
```suggestion
     *   54 bits - blknum * CHILD_BLOCK_INTERVAL * 10^5 + txindex; 54 bits represent all transactions for 85 years. (assuming CHILD_BLOCK_INTERVAL 1000 and average eth block time of 15 sec)
```

Reply:
I will add the `CHILD_BLOCK_INTERVAL`, but skip the etc block interval as the link to the exact calculations is provided ;)